### ISSUE TITLE
[Warning] compilation warning ITK_LEGACY_REMOVE

### ISSUE THREAD

**Issue by mathildemerle** [medInria/medInria-public#386]
Compilation on Ubuntu 18.04:

```
/home/mathildemerle/Dev/MUSIC3/medInria-public_ExtProjs/build/ITK/Modules/Core/Common/itkConfigure.h:82: avertissement : "ITK_LEGACY_REMOVE" redefined
 #define ITK_LEGACY_REMOVE
```

There are a lot of these warnings when you compile from scratch.

**Comment by Florent2305** [comment]
Same problem on Windows/VS2015

**Comment by Florent2305** [comment]
Closed by PR #407
